hil-test/tests/embassy_interrupt_spi_dma.rs

```diff
@@ -102,11 +128,14 @@ mod test {
                 ..Config::default()
             },
         )
+        .with_miso(miso)
```

Comment:
We didn't need data to actually be transmitted outside of the chip, so not connecting pins was fine before. Did something change?

Reply:
I thought we are still using physical pins for other SPI tests, don't we? If not, we shouldn't do it here, true

Reply:
There are no other tests here, we don't care about the input/output data, just that the whole firmware doesn't get locked up. This is why i2s would also be fine on esp32, because the peripheral works even if it sometimes outputs (or reads, unsure) 0 when it shouldn't.

I guess you are now reading back on SPI, but that is somewhat redundant with all the other spi tests we have, I think.

Reply:
I do care about reading data to detect if (for some reason) the SPI transfer ends prematurely in this scenario (when another peripheral is using DMA) - we haven't had that problem ever and I'd like to make sure we don't see that in future